.github/workflows/coverage.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
     runs-on: ubuntu-22.04
     # FIXME: disabled because the tokenless upload suddenly started to permanently fail
-    if: false # ${{ github.repository_owner == 'danmar' }}
+    #if: false # ${{ github.repository_owner == 'danmar' }}
```

Comment:
Please restore the old condition.

Reply:
my intention was to remove this if it worked.

Reply:
if it worked I would change to:

    if: ${{ github.repository_owner == 'danmar' }}

now I just want to see that it works first.